webpack/assets/javascripts/react_app/components/Layout/Navigation.js

```diff
@@ -69,17 +76,31 @@ const Navigation = ({
               groups.push({ title: sub.title, groupItems: [] });
               currIndex++;
             } else {
+              const isCurrentLocation =
+                className.includes('location-menu') &&
+                (sub.title === currentLocation ||
+                  (currentLocation === undefined &&
+                    sub.id === 'menu_item_any_location'));
+              const isCurrentOrganization =
+                className.includes('organization-menu') &&
+                (sub.title === currentOrganization ||
+                  (currentOrganization === undefined &&
+                    sub.id === 'menu_item_any_organization'));
+
               groups[currIndex].groupItems.push({
                 ...sub,
-                isActive: currentPath === sub.href?.split('?')[0],
+                isActive:
+                  currentPath === sub.href?.split('?')[0] ||
```

Comment:
when selecting an org/loc `currentPath` becomes null for a second, can you add a condition `currentPath && currentPath === sub.href?.split('?')[0])? as now when selecting an org/loc it highlights all of them for a second